src/main/java/edu/umich/ctools/sectionsUtilityTool/Friend.java

```diff
@@ -51,6 +53,7 @@
 	protected static String requesterEmailFile = null;
 	protected static String mailHost = null;
 	protected static String subjectLine = null;
+	protected static boolean mailDebug = false;
```

Comment:
This is actually Printing the raw email body to Logs, If this option is turned on will print the raw email to Logs. this Option has nothing to do with Log4J logging. From your comment i understood that this what you mean